src/test/java/uk/gov/hmcts/reform/roleassignment/controller/WelcomeControllerTest.java

```diff
@@ -57,6 +58,8 @@ void releaseDbLock() {
     @Test
     void verifyUncoveredException() {
         ResponseEntity<String> responseEntity = sut.getException("unProcessableEntity");
-        assertNull(responseEntity);
+        Assertions.assertNull(responseEntity,"");
+
+
```

Comment:
I am not sure how this will fix fortify?
I assume it should be something like 
assert responseEntity == null;